### DISCUSSION TITLE
This expression is not callable. Type 'PluginUtils' has no call signatures.ts (2349)

### DISCUSSION THREAD

**Discussion by frederikhors** [tailwindlabs/tailwindcss#10970] [Help] [answered]
**What version of Tailwind CSS are you using?**

v3.3.1 with a typescript config file.

**Describe your issue**

![image](https://user-images.githubusercontent.com/41120635/230735582-8be389c7-db0b-46e3-a84d-895aeb95d2e6.png)

Why is this error present?

The code is:

```ts
import type { Config } from "tailwindcss";

export default {
  content: [],
  theme: {
    extend: {
      borderColor: (theme) => ({
        DEFAULT: theme("colors.red.500", "currentColor"),
      }),
    },
  },
  plugins: [],
} satisfies Config;
```

**Comment by frederikhors** [comment]
Why a discussion, @adamwathan?

**Reply by adamwathan** [reply]
Hey! This isn’t a bug, theme is meant to be destructured from the object and invoking it as a function is deprecated which is why it’s not included in the types, even though it still works for now.

**Reply by mrleblanc101** [reply]
Ahhh, the documentation of vueform/multiselect is incorrect: https://github.com/vueform/multiselect?tab=readme-ov-file#styling-with-tailwind-css
![Screenshot 2025-01-13 at 2 55 11 PM](https://github.com/user-attachments/assets/b62f811f-a196-4650-9091-81b8f96c5cda)
I'll open a PR for a fix

**Comment by wongjn** [comment, marked as answer]
I believe `theme` needs to be destructured from the argument of the function:
```diff
 theme: {
   extend: {
-    borderColor: (theme) => ({
+    borderColor: ({ theme }) => ({
       DEFAULT: theme("colors.red.500", "currentColor"),
     }),
   },
```
